darwin/utils.py

```diff
@@ -67,9 +66,14 @@ def is_deprecated_project_dir(project_path: Path) -> bool:
     return (project_path / "annotations").exists() and (project_path / "images").exists()
 
 
-def get_progress_bar(array: List, description: Optional[str] = None):
-    pbar = tqdm(array)
-    pbar.set_description(desc=description, refresh=True)
+def get_progress_bar(array: List, description: str = "Progress", total: int = None):
+    try:
+        from rich.progress import track
+        pbar = track(array, description=description, total=total)
+    except ImportError:
```

Comment:
Probably we don't need this try-except clause since `tqdm` is a dependency anyways.